### DISCUSSION TITLE
Checking In

### DISCUSSION THREAD

**Discussion by semsi-dogruer** [EOT-Event-Organizer-s-Toolbox/eot-codebase#110] [General]
Hi Team! How is everyone doing? I received some feedback suggesting that having one standup a week could work well for some folks. This would be totally optional and it could actually be like a chill coffee chat more than anything.
 
I'd love to hear your opinion on this.

@0916dhkim @one2code @DaleOrders @igMike-V  @anjolaoluwaakindipe @parradaniela

**Comment by igMike-V** [comment]
I'm game.

**Comment by 0916dhkim** [comment]
Stately, the company behind xstate, does regular office hours https://www.youtube.com/playlist?list=PLvWgkXBB3dd6PXbSoctwDXjWX-AQrxhds where they share updates & take questions.

Do you think having something similar for EOT will be helpful? Not necessarily a stream, but somewhere you can bring your questions. I try to minimize the communication outside GitHub because external discussions tend to get lost, but we can arrange sessions for unblocking/clarifying (less about discussing).

**Reply by anjyakind** [reply]
That's a nice idea

**Comment by anjyakind** [comment]
I'm in support of this. As the project gets bigger we might need a place to clarify things before we start implementation. It can get quite confusing since a lot of tasks aren't fully detailed. Especially something like a guideline so that we don't constantly have merge conflicts while trying to create PR's.

**Reply by 0916dhkim** [reply]
Please ask for clarification on the issue if there's missing details. I know we pumped out a bunch of issues during the initial sprint, and they are sparse in details.

I believe it will be more beneficial to put clarification on issues than discussing them at meetings.

**Comment by semsi-dogruer** [comment]
I was thinking of having a standup/coffee chat every Wednesday at 9 a.m. EST because if I remember correctly this time slot was the most convenient for the majority of the team.

It wouldn't be mandatory and not every member would need to show up every time. If one representative from each branch (FE/BE) could make it that would be enough.

I'm open to other ideas. Also, I agree with Danny's suggestion regarding the clarification of missing details.

**Reply by 0916dhkim** [reply]
I can't do weekday 9-7 but feel free to sync without me 👍

**Comment by semsi-dogruer** [comment]
This Wednesday at 9 a.m. EST I will be in a Zoom call and anyone who has the time can join me. It will be a chill coffee chat/standup-ish type of conversation. You can find the Zoom link below.
https://us04web.zoom.us/j/74491471297?pwd=3QianpvxKf9KcCCzjayFAhXxkWkjmz.1

**Comment by semsi-dogruer** [comment]
Hey team! This Wednesday I'll be in the same Zoom call as last week at 9 a.m. EST. Hope to see you there!
https://us04web.zoom.us/j/74491471297?pwd=3QianpvxKf9KcCCzjayFAhXxkWkjmz.1

**Comment by semsi-dogruer** [comment]
Hey team! Tomorrow at 9 a.m. EST I'll be in the same Zoom call as last week. Hope to see you there!
https://us04web.zoom.us/j/74491471297?pwd=3QianpvxKf9KcCCzjayFAhXxkWkjmz.1
